### PR TITLE
feat: Esso

### PR DESCRIPTION
**Pull request by charlesdassonville**

### CODE REVIEW

**Comment by Cj-Malone** [comment]
Is this data the same as the Esso data in exxonmobil?

**Comment by charlesdassonville** [comment]
> Is this data the same as the Esso data in exxonmobil?

Not totally: exxonmobil does not return POis in France or BE. but returned in GB. 
-> i removed GB. Still FR and BE in this PR

**Comment by charlesdassonville** [comment]
@Cj-Malone can you help me with the broken pipe please ? both spiders works on my local... :/

**Comment by Cj-Malone** [comment]
CI wont run because it think more than 5 spiders have changed. https://github.com/alltheplaces/alltheplaces/actions/runs/4412809547/jobs/7732630160

File: locations/spiders/esso_fr.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    item_attributes = {
+        "brand": "Esso",
+        "brand_wikidata": "Q867662",
+    }
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+    fr_url = "https://carburant.esso.fr/api/RetailLocator/GetRetailLocations?DataSource=RetailGasStations"
+
+    def start_requests(self):
+        market_request = JsonRequest(url=self.fr_url, method="GET")
+        yield market_request
```
[the remark this answers is not in the document]
Comment (charlesdassonville) [reply locations/spiders/esso_fr.py]: you are right ! api limit to 250 -> i loop with a grid to search all pois -> i get 545 pois for france

**Comment by iandees** [comment]
Same feedback as on #4938: I would prefer that we not add pandas, and we need documentation for how you generated the H3 points.

**Pull request closed without merging**